Project/STM8S_StdPeriph_test/main.c

```diff
@@ -74,8 +76,8 @@ void main(void)
   CLK_PeripheralClockConfig(CLK_PERIPHERAL_TIMER4, ENABLE);
 
   /* Initialize LED pins in Output Mode */
-  GPIO_Init(GPIOA, (GPIO_Pin_TypeDef)(GPIO_PIN_2 | GPIO_PIN_3), GPIO_MODE_OUT_PP_LOW_FAST);
-  GPIO_WriteHigh(GPIOA, (GPIO_Pin_TypeDef)(GPIO_PIN_2 | GPIO_PIN_3));
+  GPIO_Init(GPIOH, (GPIO_Pin_TypeDef)(GPIO_PIN_2 | GPIO_PIN_3), GPIO_MODE_OUT_PP_LOW_FAST);
```

Comment:
what about using an #ifdef based on the device ? so we can keep the example working for the simple devboards